tests/console/yast2_dns_server.pm

```diff
@@ -25,12 +25,19 @@ use version_utils qw(is_leap is_sle sle_version_at_least);
 sub assert_running {
     my $self    = shift;
     my $running = shift;
+    my $cmd = '(systemctl is-active named || true) | grep -E';
 
     if ($running) {
-        assert_script_run '(systemctl is-active named || true) | grep -E "^active"', timeout => 300;
+        $cmd .= ' "^active"';
+        if (script_run $cmd) {
+            sleep 15;    # Service may take a bit of time to actually start
```

Comment:
Could you please replace this with a for-loop retrying to check for the service being active?